libs/api/transaction/feature/src/lib/api-transaction-feature.controller.ts

```diff
@@ -38,7 +39,7 @@ export class ApiTransactionFeatureController {
   @ApiBody({ type: MakeTransferRequest })
   @ApiOperation({ operationId: 'makeTransfer' })
   @ApiResponse({ type: AppTransaction })
-  makeTransfer(@Body() body: MakeTransferRequest) {
-    return this.service.makeTransfer(body)
+  makeTransfer(@Body() body: MakeTransferRequest, @Req() req: Request) {
+    return this.service.makeTransfer(body, req.ip)
```

Comment:
Let's add the whole `req` object into `makeTransfer`, and use it as the first parameter. We later need to get the `user agent` off of it too.

Reply:
![image](https://user-images.githubusercontent.com/36491/187264571-8c020db1-3a3c-49f7-929c-09be1edbd031.png)

Reply:
@RamirezAlex please make `req` the first param.